mycroft/util/parse.py

```diff
@@ -40,6 +44,27 @@ def extractnumber(text, lang="en-us"):
     return text
 
 
+def extract_datetime(text, anchorDate=None, lang="en-us"):
+
+    """Takes in a string and extracts a date and time.
+    Args:
+        text (str): the string to extract a date from
+        lang (str): the code for the language text is in
+    Return:
```

Comment:
Add doc for the anchorDate arg.

The docstring you used for the extract_datetime_en() is actually really good, pull it up to this function and I think you should be good.  The _en() version doesn't really need the full docstring since it is more of a helper function, you can just document at the extract_datetime() level.